drivers/ws281x/include/ws281x_params.h

```diff
@@ -64,6 +64,47 @@ static const ws281x_params_t ws281x_params[] =
     WS281X_PARAMS
 };
 
+/** @brief Timer used for WS281x (by the timer_gpio_ll implementation)
+ *
+ * A single timer is configured for any number of WS281x strands, so this does
+ * not need to be part of params.
+ *
+ * It is required that the timer has at least 2 channels. (Future versions may
+ * require a 3rd channel).
+ *
+ * It is required that the timer's MAX_VALUE is 2^n-1, which is a trivial but
+ * not explicitly stated case.
+ *
+ * This timer is configured at WS281x initialization time, and kept stopped
+ * outside of transmissions.
+ *
+ * The default value of 2 is chosen because the only platform on which the
+ * module is usable is nRF5x, where TIMER_DEV(1) is in use by the radio module.
+ * It is strongly advised to explicitly set this timer to a known free timer,
+ * as the default may change without notice.
+ * */
+#if !defined(WS281X_TIMER_DEV) || defined(DOXYGEN)
+#define WS281X_TIMER_DEV TIMER_DEV(2)
+#endif
+
+/** @brief Maximum value of the timer used for WS281x (by the timer_gpio_ll implementation)
+ *
+ * This macro needs to be defined to the `TIMER_x_MAX_VALUE` corresponding to
+ * the `TIMER_DEV(x)` in @ref WS281X_TIMER_DEV.
+ * */
+#ifndef WS281X_TIMER_MAX_VALUE
+#define WS281X_TIMER_MAX_VALUE TIMER_2_MAX_VALUE
+#endif
+
+/** @brief Frequency for the timer used for WS281x (by the timer_gpio_ll implementation)
+ *
+ * This should be set to a frequency that is a close multiple of 3MHz,
+ * depending on the precise low and high times. A value of 16MHz works well.
+ * */
+#ifndef WS281X_TIMER_FREQ
+#define WS281X_TIMER_FREQ 16000000
```

Comment:
```suggestion
#define WS281X_TIMER_FREQ MHZ(16)
```

reads a bit nicer

Reply:
It seems like using the MHZ macro causes some [conflicts](https://ci.riot-os.org/details/c8acd25a72374452969f4d901fd2cff0) with the ESP so I think it best to just leave it out.